src/auth/users.respository.ts

```diff
@@ -37,8 +40,35 @@ export class UsersRepository extends Repository<User> {
       }
     }
   }
+
   async getUser(idUser: string): Promise<User> {
     const user = this.findOne({ where: { id: idUser } });
     return user;
   }
+
+  async updateOne(userId: string, imagePath: string): Promise<User> {
+    try {
+      const query = await this.createQueryBuilder()
+        .update(User)
+        .set({ profileImage: imagePath })
```

Comment:
My IDE is showing me an error here:
![getUserError](https://user-images.githubusercontent.com/77193627/156228597-0e8c9272-01ac-4492-a5b9-7fb8c17e7af2.png)
You forgot a log 😉.

Reply:
hmmm , in users entity does it has profileImage column ?